### DISCUSSION TITLE
Admin Profiles

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#181] [Suggestions]
While the release of the second Permissions update is really great, I think there can be room for improvement with one more core feature - being able to create a profile for your admins.

Suggestion: On the "Additional Admins" page, in the "Admins" tab, allow for community owners to assign nicknames to admins (for the ones that don't do it themselves and just have an email to be referenced by), and link them to a player profile that can have their steam 64ID and Discord ID available.

**Comment by mvaessen** [comment]
Thanks for your suggestion!

With this update any account that doesn't have a nickname setup will be prompted to configure one. The prompts are annoying and forcing, so shortly your (active) admins should have a nickname listed. The nickname is registered on an account level, so any admin that is part of multiple teams will have the same nickname listed on all dashboards.

I agree that along with #175 there are quite some things that can make this work even better. 
For Permissions 2.1 we should definitely have an option to connect a `player` to an `admin`, so we can do all sorts of things like auto whitelisting, community website permissions in the permissions form, manage bot permissions, auto admin activation on servers etc.

**Comment by mvaessen** [comment]
This suggestion was partially implemented. The option for dashboard owners to set a nickname is not available, because this nickname is shared throughout the system. So only the admin can set their own nickname. 

In the upcoming update the owner of the dashboard can link a Player to the admin account. For now this is used to integrate the Community website and Bot permissions into the V2 permissions form.

More info: https://help.gameserverapp.com/article/248-link-a-player-to-an-admin
